src/hive/types.ts

```diff
@@ -0,0 +1,74 @@
+import { HiveApiV1 } from './api/api-v1';
+import { HiveApiV2 } from './api/api-v2';
+import { IMerjoonHttpClient } from '../common/types';
+
+export interface IHiveConfig {
+  apiKey: string;
+  httpsAgent?: IHiveConfigHttpsAgent;
+}
+
+export interface IHiveConfigHttpsAgent {
+  maxSockets?: number;
+}
+
+export enum HiveApiPath {
+  Users = 'users',
+  Projects = 'projects',
+  Actions = 'actions',
+  Workspaces = 'workspaces',
+}
+
+export interface IHiveQueryParams {
+  first: number;
+  after?: string;
+}
+
+export interface IHiveUser {
+  'id': string;
+  'fullName': string;
+  'email': string;
+}
+
+export interface IHiveProject {
+  'id': string;
```

Comment:
remove apostrophes in the keys, and add rule in the eslint for it 
should be
`id: string;`
and eslint should enforce it